### PR TITLE
[DO-1657]Create multiselect filter as Dropdown menu for PEP type column of the PEP dataset

### PR DESCRIPTION
**Pull request by AlenaYanish**

### CODE REVIEW

File: src/components/table/Table.jsx [hunk 1]
```diff
@@ -148,7 +150,7 @@ const Table = (props) => {
           </div>
         </div>
       )}
-      <div className="overflow-x-auto box">
+      <div className={`overflow-x-auto box min-h-${minHeight}`}>
```
Comment (roman-tiukh) [inline src/components/table/Table.jsx:153]: better in pixels

File: src/components/table/Table.jsx [hunk 2]
```diff
@@ -221,12 +221,14 @@ Table.propTypes = {
...
   onRowClick: PropTypes.func,
   exportUrl: PropTypes.string,
+  minHeight: PropTypes.string,
 };
 Table.defaultProps = {
   fields: [],
   axiosConfigs: {},
   onRowClick: undefined,
   exportUrl: undefined,
+  minHeight: undefined,
```
Comment (roman-tiukh) [inline src/components/table/Table.jsx:231]: default value??

File: src/components/table/useTableController.js [hunk 3]
```diff
@@ -40,7 +40,11 @@ const useTableController = (options) => {
     if (params) {
       Object.entries(params).forEach(([key, value]) => {
         if (value) {
-          urlParams.set(key, value.toString());
+          if (Array.isArray(value)) {
+            value.forEach((val) => {
+              urlParams.append(key, val);
+            });
```
Comment (roman-tiukh) [inline src/components/table/useTableController.js:46]: this code doesn't work

**Review by roman-tiukh — CHANGES_REQUESTED** [review]

File: src/components/table/useTableController.js (hunk 3 above)
Comment (AlenaYanish) [reply src/components/table/useTableController.js:46]: it depends https://github.com/3v-workspace/Data_converter/pull/1169

File: src/components/filter-fields/FilterField.jsx [hunk 4]
```diff
@@ -108,17 +109,20 @@ FilterField.propTypes = {
   filter: PropTypes.shape({
     name: PropTypes.string.isRequired,
     type: PropTypes.string.isRequired,
+    multiple: PropTypes.bool,
     placeholder: PropTypes.string,
     width: PropTypes.string,
     options: PropTypes.arrayOf(PropTypes.shape({
-      value: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
+      value: PropTypes.string,
```
[the remark this answers is not in the document]
Comment (AlenaYanish) [reply src/components/filter-fields/FilterField.jsx:116]: now, options.value is only strings

[the remark this answers is not in the document]
Comment (roman-tiukh) [reply src/components/filter-fields/FilterField.jsx:116]: but they can be numbers too

File: src/styles/tables.scss [hunk 5]
```diff
@@ -43,3 +43,6 @@
   }
 }
 
+.min-h-400 {
+  min-height: 400px;
+}
```
[the remark this answers is not in the document]
Comment (AlenaYanish) [reply src/styles/tables.scss:48]: so that a dropdown isn't truncated in the table

[the remark this answers is not in the document]
Comment (roman-tiukh) [reply src/styles/tables.scss:48]: why you create a class for this?

File: src/components/table/useTableController.js (hunk 3 above)
Comment (roman-tiukh) [reply src/components/table/useTableController.js:46]: nope, its doesn't work anyway...

File: src/components/table/Table.jsx [hunk 6]
```diff
@@ -253,12 +257,14 @@ Table.propTypes = {
   axiosConfigs: PropTypes.object,
   onRowClick: PropTypes.func,
   exportUrl: PropTypes.string,
+  minTableHeight: PropTypes.string,
```
Comment (roman-tiukh) [inline src/components/table/Table.jsx:260]:
```suggestion
  minHeight: PropTypes.string,
```

File: src/locales/en/translation.json [hunk 7]
```diff
@@ -277,7 +277,13 @@
   "pepDetailType": "PEP type",
   "pepInfo": "PEP Information",
   "pepScheme": "PEP Scheme",
-  "pepType": "PEP type",
```
Comment (roman-tiukh) [inline src/locales/en/translation.json:280]: you delete existed translation(

File: src/styles/select-dropdown.scss [hunk 8]
```diff
@@ -2,24 +2,20 @@
...
     margin-top: 0.25rem;
-    visibility: hidden;
+    display: none;
     opacity: 0;
     transition: visibility 0s linear 0.2s, opacity 0.2s 0s;
 }
 
 .select-dropdown.show {
-    visibility: visible;
+    display: block;
```
Comment (roman-tiukh) [inline src/styles/select-dropdown.scss:11]: if you change `display` only, you don't have transition
```
transition: visibility 0s linear 0.2s, opacity 0.2s 0s;
```

**Review by roman-tiukh — APPROVED** [review]

**Pull request merged**